app/models/ext_management_system.rb

```diff
@@ -718,6 +718,14 @@ def self.display_name(number = 1)
     n_('Manager', 'Managers', number)
   end
 
+  def inventory_object_refresh?
+    Settings.ems_refresh[emstype].try(:[], :inventory_object_refresh)
```

Comment:
You could use `Settings.ems_refresh.fetch_path(ems_type, :inventory_obeject_refresh)` to avoid the `try`

Reply:
:+1: done